Xamarin.Forms.Platform.Tizen/Shell/ShellNavBar.cs

```diff
@@ -29,15 +29,18 @@ public class ShellNavBar : Native.Box
 
 		bool _hasBackButton = false;
 
-		public ShellNavBar(IFlyoutController flyoutController, ShellSectionNavigation shellSectionNavigation) : base(Forms.NativeParent)
+		public ShellNavBar(IFlyoutController flyoutController) : base(Forms.NativeParent)
 		{
 			_flyoutController = flyoutController;
-			_shellSectionNavigation = shellSectionNavigation;
 
+			_menuButton = new Native.Button(Forms.NativeParent);
```

Comment:
What do you think about using `ElmSharp.Button` instead?

Reply:
Fixed.